README_SEPTENTRIO_INS_PRE_CONFIG.md

```diff
@@ -0,0 +1,42 @@
+# Pre-Config of the AsteRx-i or AsteRx-m2 dual antenna setup
+
+If Septentrio INS is selected (GPS_TYPE = 17) a pre-config of the AsteRx-i or AsteRx-m2 dual antenna setup is required. This can be done trough RX-tools or connecting the AsteRx with a com port and sending the commands over serial communication.
+
+The \n indicates a newline and is needed for the AsteRx to interpret a new command.
+
+## AsteRx-i
+
+The following commands has to be set:
+
+Define on which com-port the imu is connected to. For example com3
+    setIMUInput, COM3\n
+
+Define the imu orientation with respect to your drone setup. For default setup use the following command
+    setIMUOrientation, SensorDefault\n
+
+    If the imu is not placed with de x-axis pointing to the front of the drone and placed horizontally and upside up, use following command
```

Comment:
`de` should be `the`. The whole file should either live in the GPS folder or more realistically be a PR to the wiki once this is in.

Reply:
Yes indeed. This readme is indeed for the wiki. I tough adding this in the wike when the reviewing is done. "de" is because of my dutch.